### ISSUE TITLE
rain build --prompt fails to generate a template using Bedrock and a promt

### ISSUE THREAD

**Issue by sho-saito** [aws-cloudformation/rain#235]
Hi,

When I try to generate a template using Bedreck promt with -p flag, it fails like 

```
rain build -p "VPC with 1 private and 1 public subnet"
error invoking model: operation error Bedrock Runtime: InvokeModel, https response error StatusCode: 400, RequestID: 220aa210-e5bc-41cf-ad30-8595da0f01ab, ValidationException: The provided model identifier is invalid.
```

version:
```
Rain v1.7.5 linux/amd64
```

environment:
```
$ uname -a
Linux ip-10-0-1-164.ap-northeast-1.compute.internal 6.1.66-91.160.amzn2023.x86_64 #1 SMP PREEMPT_DYNAMIC Wed Dec 13 04:50:24 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by ericzbeard** [comment]
Is this separate from the PR you submitted?

**Comment by ericzbeard** [comment]
Oh I see, v2 is not available there but 2.1 is.. ok.

**Comment by sho-saito** [comment]
thank you for closing this and merging the PR! 🙇
